public/javascripts/SVLabel/src/SVLabel/keyboard/Keyboard.js

```diff
@@ -315,6 +320,82 @@ function Keyboard (svl, canvas, contextMenu, googleMap, ribbon, zoomControl) {
                             });
                         }
                 }
+
+                // Hotkeys for tag selection
+                var labelType = contextMenu.getTargetLabel().getProperty('labelType');
+                if (labelType == 'CurbRamp') { // Curb Ramp
+                    switch (e.keyCode) {
+                        case util.misc.getLabelDescriptions('CurbRamp')['tagInfo']['narrow']['keyNumber']: // 'a' for 'narrow'
+                            document.getElementsByClassName('context-menu-tag')[0].click();
```

Comment:
This code looks a lot better!! Is there a way to get rid of the hard-coded array indexing here?
`document.getElementsByClassName('context-menu-tag')[0].click();`

Is there a way we can get rid of the `0`?

I just want to change the number of things we have to change whenever we modify the tags

Reply:
Mmm ok I prefer the old way. It isn't great, but I'd rather not rely on a `contains` function. Just revert the change, then I will test the functionality.

Reply:
One possible other solution would be to dynamically add a class to the tag html elements on creation. (Such that a tag element would look like this: `<button class="context-menu-tag narrow-tag" ... >n<tag-underline>a</tag-underline>rrow</button>` And search by class name when clicking buttons. Since we are already performing a `.getElementsByClassName()` call the performance would probably not be that much different.

Reply:
@misaugstad I just tried out and pushed that suggestion. I think it works really well. At the cost of no added complexity, it's now easier to add new tags/ change tags in the future.